<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cipher @ypingcn</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="PlainTextWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextEdit" name="edtPlainText"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnCopyPlainText">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Copy Plain Text</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="ToolsWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="rbtnCaesar">
          <property name="text">
           <string>Caesar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbtnPlayfair">
          <property name="text">
           <string>Playfair</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbtnHill">
          <property name="text">
           <string>Hill</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labPlayfair">
          <property name="text">
           <string>Playfair's word</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPlayfair">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labHill">
          <property name="text">
           <string>Hill's secret key ( 3x3 only)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="edtHill"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnLoadFile">
          <property name="text">
           <string>Load Plain Text From File</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_1">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnEncrypt">
          <property name="text">
           <string>Encrypt --&gt;&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDecrypt">
          <property name="text">
           <string>&lt;&lt;-- Decrypt</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnAbout">
          <property name="text">
           <string>About</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="CipherTextWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextEdit" name="edtCipherText"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnCopyCipherText">
          <property name="text">
           <string>Copy Cipher Text</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
